core/dbt/tests/util.py

```diff
@@ -215,6 +215,11 @@ class TestProcessingException(Exception):
 def run_sql_with_adapter(adapter, sql, fetch=None):
     if sql.strip() == "":
         return
+
+    # make data type change if warehouse type is snowflake
+    if adapter.acquire_connection().type == "snowflake":
```

Comment:
can move back to that format, would this change the location the files would need to be kept? like pulled out of adapter level into just functional level?

Reply:
I think adapter can just copy paste this one file and do the adjustment, or import it and do the replace in the adapter

Reply:
just swapped them to csv files, and got these tests working that way, was able to remove the adapter specific logic from utils.py. so both this and snowflake version are passing, the [concurrency](https://github.com/dbt-labs/dbt-core/pull/5753) conversion is passing on core following same method but currently throwing a random syntax error on snowflake so think it might be good if all looks good.